.github/workflows/node.js.yml

```diff
@@ -26,15 +26,18 @@ jobs:
               uses: actions/setup-node@v2
               with:
                   node-version: ${{ matrix.node-version }}
-            - run: npm install
+            - run: npm i -g npm@7
```

Comment:
I thought the inconsistencies with CI was because of the `npm` version. Turns out the inconsistencies were just because some of the jobs weren't running `npm ci`.